src/Player.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Created by strejolopez on 11/29/16.
+ */
+public class Player {
+    private String name;
+    private String token;
+    private int money;
+
+    public Player(String name, String token) {
+        this.name = name;
+        this.token = token;
+        this.money = 1500;
+    }
+
+    public String getName() { return this.name; }
+    public String getToken() { return this.token; }
+    public int getMoney() { return this.money; }
+
+
+    public void payMoney(int amount){
```

Comment:
As discussed this method has some terrifying tabs.  Clean them up a bit before pushing.